api-report/sequence.api.md

```diff
@@ -44,12 +44,13 @@ import { ReferencePosition } from '@fluidframework/merge-tree';
 import { ReferenceType } from '@fluidframework/merge-tree';
 import { Serializable } from '@fluidframework/datastore-definitions';
 import { SharedObject } from '@fluidframework/shared-object-base';
+import { SlidingPreference } from '@fluidframework/merge-tree';
 import { SummarySerializer } from '@fluidframework/shared-object-base';
 import { TextSegment } from '@fluidframework/merge-tree';
 import { TypedEventEmitter } from '@fluidframework/common-utils';
 
 // @public
-export type CompressedSerializedInterval = [number, number, number, IntervalType, PropertySet];
+export type CompressedSerializedInterval = [number, number, number, IntervalType, PropertySet, IntervalStickiness] | [number, number, number, IntervalType, PropertySet];
```

Comment:
The breaking change tool calls this a breaking change, though I'm not sure if it violates our guarantees or if we care about this type.

This isn't typed as `IntervalStickiness | undefined` as `JSON.stringify([undefined])` would return `"[null]"`, which defeats the purpose of not including the stickiness

Reply:
ick. we should do the work to hide the `IntervalCollection` class from the public API in next so that we can also hide this and other serialized format types from the public API. this should have also been marked internal as-is (which IMO we shouldn't bother fixing in next--not exposing at all is the better fix).

Reply:
could you file some work to do that in a followup?

Reply:
Created [4494](https://dev.azure.com/fluidframework/internal/_workitems/edit/4494) to track this